Makefile

```diff
@@ -442,6 +442,11 @@ generate-docs: hugo ## Re-generate docs directory from the website directory
 	cd website && npm install
 	$(HUGO_PATH)/hugo -s website -d ../docs
 
+.PHONY: run-docs
+run-docs: 
```

Comment:
run-docs: hugo (check if we have hugo and get HUGO_PATH variable)
I think we need it here because this goal can be run separately